FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Fernando De La Rosa on 08/09/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+final class FeedImageMapper {
+	private struct Root: Decodable {
+		private let items: [Image]
+
+		internal var feedImages: [FeedImage] {
```

Comment:
The convention in this codebase is to omit the default access level keyword `internal`, please remove it.
```suggestion
		var feedImages: [FeedImage] {
```

Reply:
Done